app/js/lib/common/errors.js

```diff
@@ -0,0 +1,8 @@
+export function HttpError(statusCode, message) {
```

Comment:
also, here is what I did for my SpeechSynthesis code: https://github.com/fxbox/calendar/compare/master...julienw:speak-notifications?expand=1#diff-d66595dd89af02cf3768c786b295f8dcR1